homeassistant/components/airtouch5/cover.py

```diff
@@ -0,0 +1,139 @@
+"""Representation of the Damper for AirTouch 5 Devices."""
+import logging
+from typing import Any
+
+from airtouch5py.airtouch5_simple_client import Airtouch5SimpleClient
+from airtouch5py.packets.zone_control import (
+    ZoneControlZone,
+    ZoneSettingPower,
+    ZoneSettingValue,
+)
+from airtouch5py.packets.zone_name import ZoneName
+from airtouch5py.packets.zone_status import ZoneStatusZone
+
+from homeassistant.components.cover import (
+    ATTR_POSITION,
+    CoverDeviceClass,
+    CoverEntity,
+    CoverEntityFeature,
+)
+from . import Airtouch5ConfigEntry
+from homeassistant.core import HomeAssistant, callback
+from homeassistant.helpers.device_registry import DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+
+from .const import DOMAIN
+from .entity import Airtouch5Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: Airtouch5ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up the Airtouch 5 Cover entities."""
+    client = config_entry.runtime_data
+
+    entities: list[CoverEntity] = []
+
+    # Each zone has a cover for its open percentage
+    for zone in client.zones:
+        entities.append(
+            Airtouch5ZoneOpenPercentage(
+                client, zone, client.latest_zone_status[zone.zone_number].has_sensor
+            )
+        )
+
+    async_add_entities(entities)
+
+
+class Airtouch5ZoneOpenPercentage(CoverEntity, Airtouch5Entity):
+    """How open the damper is in each zone."""
+
+    _attr_device_class = CoverDeviceClass.DAMPER
+
+    def __init__(
+        self, client: Airtouch5SimpleClient, name: ZoneName, has_sensor: bool
+    ) -> None:
+        """Initialise the Cover Entity."""
+        super().__init__(client)
+        self._name = name
+
+        self._attr_unique_id = f"zone_{name.zone_number}_open_percentage"
+        self._attr_name = "Damper"
```

Comment:
What's a damper exactly?

Reply:
The [damper](https://en.wikipedia.org/wiki/Damper_(flow)) is how each zone controls how much air is allowed through it. It can be fully open, fully closed, or anywhere in between.

Reply:
AirTouch5 is a multi-zone AC. There is a central AC unit with ducts going out to each zone (usually a room), each of these ducts has a damper inline, the damper controls the rate of flow of air in to that area.
[Damper example image](https://media.angi.com/s3fs-public/locating-HVAC-dampers.png?impolicy=infographic)

Reply:
Too fast 😅

Reply:
Honestly, maybe a `valve` would also fit here. But I'll leave that up to you two

Reply:
Oh never mind, apparently we have a device class specific to damper, so cover is the best I would assume